cmake/modules/DarwinSDKs.cmake

```diff
@@ -14,7 +14,7 @@ set(SUPPORTED_TVOS_ARCHS "arm64")
 set(SUPPORTED_TVOS_SIMULATOR_ARCHS "x86_64;arm64")
 set(SUPPORTED_WATCHOS_ARCHS "armv7k")
 set(SUPPORTED_WATCHOS_SIMULATOR_ARCHS "i386;arm64")
-set(SUPPORTED_OSX_ARCHS "x86_64")
+set(SUPPORTED_OSX_ARCHS "x86_64;arm64;arm64e")
```

Comment:
This shouldn't need to be committed, it is a local workaround. Hopefully there is a new CMake release soon with the fix: https://gitlab.kitware.com/cmake/cmake/-/merge_requests/5023